### PR TITLE
Fix/win genesis import

### PR DESCRIPTION
**Pull request by oleksiylukin**
Implementation of genesis loading aligned to config loading strategy

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/appdata/impl/TwoFactorAuthServiceImpl.java [hunk 1]
```diff
@@ -48,7 +48,9 @@ public TwoFactorAuthServiceImpl(@Named("DBRepository") TwoFactorAuthRepository r
     public TwoFactorAuthServiceImpl(TwoFactorAuthRepository repository, String issuerSuffix, Random random,
                                     TwoFactorAuthRepository targetFileRepository) {
         if (issuerSuffix == null || issuerSuffix.trim().isEmpty()) {
-            issuerSuffix = RuntimeEnvironment.getInstance().isDesktopApplicationEnabled() ? "desktop" : "web";
+          //TODO: what is the difference?  
+          //  issuerSuffix = RuntimeEnvironment.getInstance().isDesktopApplicationEnabled() ? "desktop" : "web";
```
Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/appdata/impl/TwoFactorAuthServiceImpl.java:52]: Issuer suffix required for 2fa qr codes, which will create two separate entries in Google Authenticator for same account and will not replace the existing, when 2fa was enabled on desktop, but was already enabled on web before

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/RuntimeEnvironment.java [hunk 2]
```diff
@@ -62,14 +61,6 @@ void setup() {
             b = true;
         }
         isHeadless = b;
-        try {
-            Class.forName("javafx.application.Application");
```
Comment (AndrewBoyarsky) [inline apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/RuntimeEnvironment.java:66]: This check is needed for 2fa app mode detection

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/UserResourceLocator.java [hunk 3]
```diff
@@ -1,74 +1,76 @@
...
+    }
+
+    @Override
+    public Optional<InputStream> locate(final String resourceName) {
+        Objects.requireNonNull(resourceName);
+        return locateInDirectories(resourceName).or(() -> locateInResources(resourceName));
+    }
+
+    private Optional<InputStream> locateInResources(String resourceName) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
```
Comment (yuriy-larin-firstbridge) [inline apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/UserResourceLocator.java:40]: Check : `Objects.requireNonNull(resourceName);` would be good here.

File: apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/UserResourceLocator.java [hunk 4]
```diff
@@ -1,74 +1,76 @@
...
+        InputStream is = classloader.getResourceAsStream(path);
+        if (is != null) {
+            log.info("Located in resources, resource={}", path);
+        }
+
+        return Optional.ofNullable(is);
+    }
+
+    private Optional<InputStream> locateInDirectories(String resourceName) {
+        List<String> searchDirs = new ArrayList<>();
```
Comment (yuriy-larin-firstbridge) [inline apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/env/config/UserResourceLocator.java:51]: Check : `Objects.requireNonNull(resourceName);` would be good here.

**Review by yuriy-larin-firstbridge — COMMENTED** [review]
Minor suggestions

**Pull request merged**